apps/charterafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,227 @@
+
+const Footer = {
+  slug: "footer",
+  label: {
+    en: "Footer",
+    fr: "Bas de page",
+    pt: "Rodapé",
+  },
+  fields: [
+    {
+      name: "siteDescription",
+      label: {
+        en: "Site Description",
+        fr: "Description du site",
+        pt: "Descrição do Site",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    {
+      name: "projectDescription",
+      label: {
+        en: "Project Description",
+        pt: "Descrição do Projeto",
+        fr: "Description du projet",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    linkGroup(),
+    {
+      name: "newsletter",
+      type: "group",
+      label: {
+        en: "Newsletter",
+        fr: "Newsletter",
+        pt: "Boletim de Notícias        ",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "embedCode",
+          type: "code",
+          label: {
+            en: "Embed Code",
+            fr: "Code d'intégre",
+            pt: "Código Incorporado",
+          },
+          localized: true,
+          required: true,
+        },
+      ],
+    },
+    {
+      name: "logo",
+      required: true,
+      localized: true,
+      type: "group",
+      label: {
+        en: "Footer Logo",
+        pt: "Imagem",
+        fr: "Logo de pied de page",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Footer Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "src",
+          type: "upload",
+          relationTo: "media",
+          required: true,
+          localized: true,
+          filterOptions: {
+            mimeType: { contains: "image" },
+            mimeTypes: ["images/*"],
+          },
+          label: {
+            en: "Footer Logo",
+            pt: "Imagem",
+            fr: "Logo de pied de page",
+          },
+        },
+      ],
+    },
+    {
+      type: "group",
+      name: "contact",
+      label: {
+        en: "Contact",
+        pt: "Contact",
+        fr: "Contact",
+      },
+      admin: {
+        width: "100%",
+      },
+      fields: [
+        {
+          name: "email",
+          type: "email",
+          label: {
+            en: "Email",
+            pt: "Email",
+            fr: "Email",
+          },
+          admin: {
+            width: "50%",
+          },
+        },
+        {
+          name: "twitter",
```

Comment:
What are we asking for i.e. handle or URL? Either update the label or add [description](https://payloadcms.com/docs/fields/overview#description)